devops/azure-pipelines/gradle-build-test-stage-template.yml

```diff
@@ -0,0 +1,20 @@
+steps:
+- task: SonarCloudPrepare@1
+  inputs:
+    SonarCloud: 'Forge-SonarCloud'
+    organization: '1058-sophia-gavrila-forge'
+    scannerMode: 'Other'
+    extraProperties: |
+      sonar.branch.name=main
```

Comment:
don't hardcode the branch name like this - it's already wrong for the auto-on PR builds.
sonar scanner discovers the branch name automatically with its git and github integrations, so just let it be.